aten/src/ATen/CPUApplyUtils.h

```diff
@@ -59,8 +59,11 @@ struct strided_tensor_iter_fixed {
   T* data_ = NULL;
   int64_t dim_ = 0;
 
+  // NOLINTNEXTLINE(*array*)
   int64_t counter_[N] = {0};
```

Comment:
These could be std::arays, couldn't they?

Reply:
counter_ etc, are public members,  I'm afraid changing them to std::array will affect pointer code.